CMSgov/onemac
# Oy2 21563

Story: https://qmacbis.atlassian.net/browse/OY2-21563
Endpoint: See github-actions bot comment

### Details

Add user who did withdrawl to emails.

### Changes

- Pass in user already retrieved for security into the email functions.

### Test Plan

1. Login as a statesubmitter
2. Navigate to package view and find a package which the current user did not submit
3. Withdraw the package (from the three dots action bar)
4. Verify the logged in user appears in the email for both state and CMS
